spec/config/initializers/filter_parameter_logging_spec.rb

```diff
@@ -1,12 +0,0 @@
-# frozen_string_literal: true
```

Comment:
Again, it was an intentional decision to drop the query logging and this test is here to ensure that keeps happening.  It'll need to be added back.